ui/components/Header.js

```diff
@@ -527,11 +527,16 @@ class Header extends React.Component {
   }
   componentDidMount() {
     this._isMounted = true;
-    const brokerStatusSub = subscribeBrokerStatusEvents(data => {
-      console.log({ brokerData : data })
-      this.setState({ brokerStatus : data?.subscribeBrokerConnection })
-    });
-    this.setState({ brokerStatusSubscription : brokerStatusSub })
+    const checkServiceStatus = () => {
+      const brokerStatusSub = subscribeBrokerStatusEvents(data => {
+        console.log({ brokerData : data });
```

Comment:
Isn't it creating a new subscription every 5 sec? In case it is, the previous subscription should be flushed down. Addionally, the health check is great way of maintaining the correct UI status wrt to connectivity to broker.

Reply:
It shouldn't be, it should be checking to see if it is active, not creating a subscription.

We just need a healthcheck, because the only time you're seeing this is during Docker build. I can't test this locally, which I haven't figured out the easiest way to do this.

Reply:
https://github.com/meshery/meshery/blob/0c4bc75fda98985607735ce9b521653951bf22bd/ui/components/graphql/subscriptions/BrokerStatusSubscription.js

As it written, I think that it is creating a subscription to the grapqhQL query `BrokerStatusSubscription`. If we don't flush the previous subscription data then we will be in loop. 

Although if you're saying it isn't I will confirm this after checking out to your PR in a while. I may be wrong here.

Reply:
I can test it again too, I can revert part of it to the original, and just keep the health check portion.

Reply:
After some testing, I don't see the subscription being recreated.

The lines of code are simply moved into their own function so that when the health check for the subscription if it is live or not, it will ping for this every 5 minutes. Which should give enough time to load from the Docker build.